docs/DafnyRef/Modules.md

```diff
@@ -453,6 +453,25 @@ in `B` nor as `Z.a` in C would be valid, because `a` is not in `Z`.
 The default export set is important in the resolution of qualified
 names, as described in [Section 4.8](#sec-name-resolution).
 
+There are a few unusual cases to be noted:
+- an export set can be completely empty, as in `export Nothing`
+- an anonymous export set can be completely empty, as in `export`
```

Comment:
Is this a way to make the module entirely private? If so, you could mention it.

Reply:
I don't like the term "anonymous export set". It _does_ have a name, even if you don't give the name explicitly. (This is unlike a `constructor` in a `class`, which can indeed be anonymous.) If you omit the name of the export set, the name of the export set defaults to the name of the module. Therefore, "eponymous export set" is more appropriate.

For example, consider

``` dafny
module Module {
  export
    provides F
  export Friends extends Module
    provides G

  function F(): int
  function G(): int
}
```

Here, the `Friends` export set extends the eponymous export set. Here, it is important that the first export set does have a name.

A client can use any of these import declarations:

``` dafny
import Module
import Module`Module
import Module`Friends
```

where the first two mean the same thing. (Each one of these three introduces the local name `Module`, so if a client wants to use more than one of these imports, the client needs to provide an explicit local name.)

Reply:
Reworded